### PR TITLE
Include day of week effect in pipeline analysis

### LINKED ISSUE #257: Include day of week effect in pipeline analysis
This looks great!

Wrt using `softmax`, I think that would be cool after we clear #251 e.g. we might model a weekly process but then broadcast out the weekly observations to a day of the week in a way that respects the weekly average obs. 

This implementation hit a different form of modelling and looks nice! 

Note: this was part of analysis plan and should now be added to pipeline.

_Originally posted by @SamuelBrand1 in https://github.com/CDCgov/Rt-without-renewal/pull/253#pullrequestreview-2096131916_
            
As per title in light of this PR

### PR DESCRIPTION
**Pull request by SamuelBrand1**
Small PR which implements the day of week observation effect introduced by #253 and #258  to the pipeline generative model. Also removes older "truth data" sims.

Closes #257 .

### CODE REVIEW

**Review by seabbs — CHANGES_REQUESTED** [review]
This looks good but has there been a git history issue here? Looks like a bunch of changes I thought we had merged already. 

Also looks like some files have been committed accidentally. Suggest using a .gitignore in truth data and adding these file types + getting rid of the placeholder file. Or do we want to commit this data as fixes the sims for onwards analysis?

File: EpiAware/src/EpiLatentModels/HierarchicalNormal.jl [hunk 1]
```diff
@@ -35,7 +35,7 @@ Generate latent variables from the hierarchical normal distribution.
 "
 @model function EpiAwareBase.generate_latent(obs_model::HierarchicalNormal, n)
     std ~ obs_model.std_prior
-    ϵ_t ~ MvNormal(I(n))
+    ϵ_t ~ MvNormal(n, one(eltype(std)))
```
[the remark this answers is not in the document]
Comment (SamuelBrand1) [reply EpiAware/src/EpiLatentModels/HierarchicalNormal.jl:38]: Commit history

**Comment by SamuelBrand1** [comment]
> This looks good but has there been a git history issue here? Looks like a bunch of changes I thought we had merged already.

I started this branch before these changes were merged to main and squashed them into this branch.

**Comment by seabbs** [comment]
That doesn't sound like the right git flow to me? I've rebase so maybe will fix

**Review by seabbs — COMMENTED** [review]
Rebased fixed the issue with old stuff here but commited files are still here. Are they meant to be?

**Comment by SamuelBrand1** [comment]
> Rebased fixed the issue with old stuff here but commited files are still here. Are they meant to be?

I don't see any committed files except some placeholder.md files?

**Review by seabbs — APPROVED** [review]

**Pull request merged**